app/code/community/Sendsmaily/Sync/Helper/Data.php

```diff
@@ -184,17 +291,17 @@ public function gatherCustomerExtra($customerId, $fields)
    * @param int $limit Limit batch size.
    * @return array Unsubscriber emails list.
    */
-  public function getUnsubscribersEmails($limit)
+  public function getUnsubscribersEmails($limit = 1000)
   {
     $curl = Mage::getModel('sync/curl');
     $unsubscribersEmails = array();
     $data = array(
         'list' => 2,
         'limit' => $limit,
+        'offset' => 0,
```

Comment:
Found a bug that the `$offset` was never given an initial value. Incrementing null gives 1 but that is not a solution.